ui-tests/tests/completer.test.ts

```diff
@@ -41,3 +46,42 @@ test('test complete updates cell', async ({ page }) => {
 
 });
 
+test('lower case completion', async ({ page }) => {
+  // Create a new Notebook
+  await page.menu.clickMenuItem('File>New>Notebook');
+  await page.click('button:has-text("Select")');
+
+  // Wait until kernel is ready
+  await page.waitForSelector(
+    '#jp-main-statusbar >> text=Python 3 (ipykernel) | Idle'
+  );
+
+  // type 'sel' in the first cell
+  await page.notebook.enterCellEditingMode(0);
+  await page.keyboard.type('sel');
+
+  await page.keyboard.press('Tab');
+  const suggestions = page.locator('.jp-Completer');
+  await expect(suggestions).toBeVisible();
+  await expect(suggestions.locator('code:text-is("SELECT")')).toHaveCount(1);
+});
+
+test('in-word suggestion', async ({ page }) => {
+  // Create a new Notebook
+  await page.menu.clickMenuItem('File>New>Notebook');
+  await page.click('button:has-text("Select")');
+
+  // Wait until kernel is ready
+  await page.waitForSelector(
+    '#jp-main-statusbar >> text=Python 3 (ipykernel) | Idle'
+  );
+
+  // type 'se' in the first cell
+  await page.notebook.enterCellEditingMode(0);
+  await page.keyboard.type('se');
+
+  await page.keyboard.press('Tab');
+  const suggestions = page.locator('.jp-Completer');
+  await expect(suggestions).toBeVisible();
+  await expect(suggestions.locator('code:text-is("INSERT INTO")')).toHaveCount(1);
+});
```

Comment:
these tests look pretty similar - I think playwright allows you to [parametrize](https://playwright.dev/docs/test-parameterize) tests to avoid duplicated code

Reply:
Done, thanks for suggestion.